### PR TITLE
[AC-8371] Build deferrable modal interface 

### PR DESCRIPTION
**Pull request by iankigen**
## [AC-8371](https://masschallenge.atlassian.net/browse/AC-8371)

**Changes introduced**
- update dimmer background-color

[Sibling PR](https://github.com/masschallenge/accelerate/pull/2921)

[Ref](https://github.com/masschallenge/accelerate/pull/2921#discussion_r596248986):
>hfreund 
@iankigen I did not see the dimmed background on test2

 
>iankigen
it is there, however, the background is set to transparent in sui-theme override.
https://test2.masschallenge.org/expert/mentor-commitments/ - you should not be able to click on anything outside the modal

 
>hfreund
I see. It looks like we did that specifically for Office Hours. Looking back at that and in light of how we're using modals now, I think we should make the background more distinct. Could you update the override to:
background-color: rgba(0,0,0,.5);

### CODE REVIEW

**Review by hfreund — APPROVED** [review]
✅ Code changes are good
✅ Checked that the modal has a dark transparent background
✅ Checked the office hours interface where the modal is also used and verified that it looks good (subjective, but opinions count here 😄 )

File: src/themes/masschallenge/modules/dimmer.overrides [hunk 1]
```diff
@@ -1,3 +1,3 @@
 .ui.page.modals.dimmer.transition.visible.active {
-  background-color: transparent;
+  background-color: rgba(0,0,0,0.5);
```
Comment (geofrocker) [inline src/themes/masschallenge/modules/dimmer.overrides:2]: why are we hard coding this?

**Review by geofrocker — CHANGES_REQUESTED** [review]

File: src/themes/masschallenge/modules/dimmer.overrides (hunk 1 above)
Comment (iankigen) [reply src/themes/masschallenge/modules/dimmer.overrides:2]: This is meant to change the background of dimmer from `transparent` to `rgba(0,0,0,0.5)`
> Looking back at that and in light of how we're using modals now, I think we should make the background more distinct. Could you update the override to:
background-color: rgba(0,0,0,.5);

**Review by geofrocker — APPROVED** [review]
Looks good

**Pull request merged**